### DISCUSSION TITLE
CC2531 - Problem with pairing; first installation

### DISCUSSION THREAD

**Discussion by gmo4u** [Koenkk/zigbee2mqtt#9950] [Devices]
I try zigbee2mqtt with HA first time.

Home Assistant OS 6.6
Zigbee2mqtt Current version: 1.22.0-3
coordinator CC2531 with 20211115 (https://github.com/Koenkk/Z-Stack-firmware/raw/master/coordinator/Z-Stack_Home_1.2/bin/default/CC2531_DEFAULT_20211115.zip)

For 2 weeks I can't start. First I try CC2531 with soft 20201128 now with newest. Both cases this same problem.
I have USB cabel. 
I have default configuration without any changes (I was try with user, pass, serrver for mqtt). Everythink looks good.

```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] socat.sh: executing... 
[08:27:56] INFO: Socat not enabled, marking service as down
[cont-init.d] socat.sh: exited 0.
[cont-init.d] zigbee2mqtt.sh: executing... 
[08:27:59] INFO: MQTT available, fetching server detail ...
[08:27:59] INFO: MQTT server settings not configured, trying to auto-discovering ...
[08:28:01] INFO: Configuring 'mqtt://core-mosquitto:1883' mqtt server
[08:28:02] INFO: MQTT credentials not configured, trying to auto-discovering ...
[08:28:03] INFO: Configuring'addons' mqtt user
[08:28:03] INFO: Previous config file found, checking backup
[08:28:03] INFO: Creating backup config in '/config/zigbee2mqtt/.configuration.yaml.bk'
[08:28:03] INFO: Adjusting Zigbee2mqtt core yaml config with add-on quirks ...
[cont-init.d] zigbee2mqtt.sh: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[08:28:06] INFO: Handing over control to Zigbee2mqtt Core ...
> zigbee2mqtt@1.22.0 start
> node index.js
Zigbee2MQTT:warn  2021-12-02 08:28:20: Using experimental new availability feature
```

then I try pairing with diferent controllers

```
Zigbee2MQTT:debug 2021-12-02 08:40:47: Received MQTT message on 'zigbee2mqtt/bridge/request/permit_join' with data '{"device":null,"time":254,"transaction":"0d622-1","value":true}'
Zigbee2MQTT:info  2021-12-02 08:40:47: Zigbee: allowing new devices to join.
Zigbee2MQTT:info  2021-12-02 08:40:48: MQTT publish: topic 'zigbee2mqtt/bridge/response/permit_join', payload '{"data":{"time":254,"value":true},"status":"ok","transaction":"0d622-1"}'
```

and nothing...

Controlers works fine - they were check.
I was try whith zigbee2mqtt on rasbian and i had this same.

**Comment by pwl** [comment]
I have the exact same problem on a fresh install of Home Assistant OS 7.0, with Zigbee2MQTT version 1.22.1-1, installed through the supervisor. I'm also using the CC2531 module with 20201127 firmware.

My logs seem to be exactly the same as @gmo4u , here's the [full log](https://gist.github.com/pwl/3330cee5010194204f26fda01be0c63c).

The "Devices" section of the web ui seems to hang on "Loading, please wait...", the rest of the ui seems to work fine. 

![image](https://user-images.githubusercontent.com/72500/145182772-5014c3e4-7d34-4783-a6f9-9cf36aa91c62.png)

**Reply by gmo4u** [reply]
I was change USB CABLE and USB port on PI on nr4 and this was help. 2 weeks work ;)

**Reply by pwl** [reply]
I tried changing the ports on my pi but nothing helped. I'm happy it at least worked out for you.

**Reply by gmo4u** [reply]
From the begining i doesnt have Loading... and i see some comunication in your log. I haven't.
I tried:
1. Rasbian & Zigbee2MQTT to see it is or not problem HA (https://www.zigbee2mqtt.io/guide/installation/01_linux.html#determine-location-of-the-adapter-and-checking-user-permissions)
2. Ubuntu on virtualmachine on my computer to see it is problem my PI or not (and it worked). (this same instalation)
And ofcourse 3 times install HA ;)
